docs/Tutorials/Basics/simple-drops.md

```diff
@@ -0,0 +1,225 @@
+
+### The NEAR-API-JS Approach
+
+With NEAR-API-JS, these two tasks are seperate, as the `create_drop` function only creates the drop. The keys that are passed into it must be generated seperately.These two tasks are separate and pre-created keys *must* be passed into create_drop
+
+:::tip
+Note that with NEAR-API-JS, an attached deposit must be added to the `functionCall`. This is to cover the cost of creating the drop. With simple drops, the cost is the sum of $NEAR embedded in each link as well as the storage costs on the protocol.
+
+In these tutorials, a generous deposit is attached and any unused $NEAR is simply added to the funder's [Keypom balance](../../Concepts/Keypom%20Protocol/balances.md)
+:::
+
+<Tabs>
+<TabItem value="KPJS" label="🔑Keypom-JS SDK">
+
+```js reference
+https://github.com/keypom/keypom-js/blob/49244d227f23535ae8962707183d1eca59280d29/docs-examples/keypom-js-sdk/simple-example.js#L27-L41
+```
+
+</TabItem>
+<TabItem value="NRJS" label="💻NEAR-API-JS">
+
+```js reference
+https://github.com/keypom/keypom-js/blob/49244d227f23535ae8962707183d1eca59280d29/docs-examples/near-api-js/simple-near-example.js#L26-L51
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+All function parameters and default values for the SDK and Keypom functions can be found in the [SDK Typedocs](../../keypom-sdk/modules.md). For NEAR-API-JS functions, their arguments can be found [here](https://docs.near.org/tools/near-api-js/reference)
+:::
+
+---
+
+## Full Solution
+The final code, using the skeleton defined in the [introduction](simple-drops.md#introduction) to create a simple 1 $NEAR linkdrop can be seen below.
+
+<Tabs>
+<TabItem value="KPJS" label="🔑Keypom-JS SDK">
+
+```js reference
+https://github.com/keypom/keypom-js/blob/49244d227f23535ae8962707183d1eca59280d29/docs-examples/keypom-js-sdk/simple-example.js#L1-L55
+```
+
+</TabItem>
+<TabItem value="NRJS" label="💻NEAR-API-JS">
+
+```js reference
+https://github.com/keypom/keypom-js/blob/49244d227f23535ae8962707183d1eca59280d29/docs-examples/near-api-js/simple-near-example.js#L1-L53
+```
+
+</TabItem>
+</Tabs>
+
+---
+
+## Testing
+### Running the Script
+Here, you'll learn how to run the code that was just covered, and what to expect.
+
+To view the completed code, clone the following repo:
+``` bash
+git clone https://github.com/keypom/keypom-js && cd keypom-js/docs-examples/keypom-js-sdk/simple-example.js
+```
+To run the code you just cloned, install all the necesasry packages. 
+```bash
+npm install
+```
+
+:::caution
+Prior to running these scripts, ensure you replace all instances of `keypom-docs-demo.testnet` in the script with the credentials of your account found in your `~/.near-credentials` folder
+:::
+
+From there, you are able to run this Simple Drop script that was made in this tutorial using the following command:
+``` bash
+npm run simple-keypom
+```
+:::note
+The SDK script is being tested here; use `npm run simple-near` to test the NEAR-API-JS script instead.
```

Comment:
Let's remove this since it doesn't make sense to run the NAJ script as it doesn't offer anything different.

Reply:
if we're offering the NAJ code blocks, shouldn't we also be giving them instructions on how to run them? Otherwise, they'd need to figure out the file layout and manually run it without the shortcut